src/components/paginationProfile.vue

```diff
@@ -1,42 +1,42 @@
 <template>
-    <div>
-      <nav aria-label="Page navigation example">
```

Comment:
sound like unecessary changes. please use prettier to format code as it was used in the repository.

Reply:
This file is unnecessary. Nothing was changed.